packages/evo-marko/src/tags/evo-icon-button/index.marko

```diff
@@ -0,0 +1,68 @@
+static const VALID_SIZES = ["small", "large"];
+import type { AttrString } from "marko/tags-html";
+static var validPriorities = ["primary", "secondary", "tertiary"];
+import type { Input as ButtonInput } from "../evo-button/index.marko";
+import type { Input as BadgeInput } from "../evo-badge/index.marko";
+
+export interface Input extends Omit<Marko.HTML.Button, "type">, Omit<Marko.HTML.A, "type" |`on${string}`> {
+  badge?: Marko.AttrTag<BadgeInput>;
+  badgeNumber?: number | string;
+  href?: string;
+  transparent?: boolean;
+  priority?: "primary" | "secondary" | "tertiary" | "none";
+  size?: "small" | "large";
+  partiallyDisabled?: ButtonInput["partiallyDisabled"];
+  onEscape?: ButtonInput["onEscape"];
```

Comment:
You omitted `"type"` from _both_ `button` and `a`, so it was lost. One option is to add it back like this (I'm not sure why TS didn't just do this by default with no `Omit`)


```
  type?: Marko.HTML.Button["type"] & Marko.HTML.A["type"]
```